src/dialogs/PreviewUploadOverlay.cpp

```diff
@@ -43,11 +43,18 @@ PreviewUploadOverlay::PreviewUploadOverlay(QWidget *parent)
         vlayout->setSpacing(conf::modals::WIDGET_SPACING);
         vlayout->setMargin(conf::modals::WIDGET_MARGIN);
 
+
```

Comment:
You should run `make lint` in your source folder, since this will make the linter complain.